pb-coroutines-kafka-retry/src/main/kotlin/io/provenance/kafka/coroutines/retry/store/InMemoryConsumerRecordStore.kt

```diff
@@ -28,11 +28,12 @@ fun <K, V> inMemoryConsumerRecordStore(
 
     override suspend fun putOne(
         item: ConsumerRecord<K, V>,
+        e: Throwable?,
         mutator: (RetryRecord<ConsumerRecord<K, V>>) -> RetryRecord<ConsumerRecord<K, V>>
     ) {
         val record = getOne(item)
         if (record == null) {
-            data += RetryRecord(item, 0, OffsetDateTime.now()).also {
+            data += RetryRecord(item, 0, OffsetDateTime.now(), e?.message ?: "No error message available").also {
```

Comment:
Maybe it would make more sense using `.orEmpty()` here?

Reply:
That works for sure, will make that change